website/content/partials/telemetry-metrics/vault/raft/delete.mdx

```diff
@@ -0,0 +1,5 @@
+### vault.raft.delete ((#vault-raft-delete))
```

Comment:
I don't think we emit this metric anymore (if we ever did).

Reply:
There's a raft_storage.fsm.delete metric, but I'm not sure we ever actually emit that in practice, will follow up once I've done more research.

Reply:
I deleted the partial and removed it from the relevant index pages. Let me know if we should also remove  raft_storage.fsm.delete